server/core/src/https/v1_oauth2.rs

```diff
@@ -200,6 +200,11 @@ pub(crate) async fn oauth2_id_scopemap_post(
     Json(scopes): Json<Vec<String>>,
 ) -> Result<Json<()>, WebError> {
     let filter = oauth2_id(&rs_name);
+
```

Comment:
Yeah, I think we don't need this error here, if you set an empty scope map via the api, you're saying "it doesn't need to be set". I think just the changes to valueset suffice here since the cli has some other ux guards now.

Reply:
Did we want to delete the entire delete endpoint then as a delete would become a empty update? Depricate the api and switch over the cli?